src/onboarding/family/familyOverview/FamilyOverview.tsx

```diff
@@ -0,0 +1,109 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { FlatList, View } from 'react-native';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { getAvatarByNameOrDefault } from '../../../_assets/images';
+import { Icon } from '../../../_components';
+import { useOnboarding } from '../../../_context';
+import { TMainNavigationProp } from '../../../_routing';
+import { useGetFamilyMembers } from '../_queries/useGetFamilyMembers';
+import * as Styled from './style';
+
+type TProps = {
+  navigation: TMainNavigationProp<'Profile'>;
+};
+
+enum TFormItemType {
+  FamilyMemberItem,
+  AddFamilyMemberButton,
+  EmptyItem,
+}
+
+export const FamilyOverview = ({ navigation }: TProps) => {
+  const { data: familyMembers } = useGetFamilyMembers();
+  const me = useMemo(() => {
+    return familyMembers?.filter(({ mainFamilyMember }) => mainFamilyMember)?.[0];
+  }, [familyMembers]);
+  const formItems = useMemo(() => {
```

Comment:
is this necessary? I think you can do the check inline instead of in a separate function, then the enum is also not necessary